src/app/shared/logger.service.ts

```diff
@@ -1,4 +1,4 @@
-import { Injectable } from '@angular/core';
+import { Injectable } from "@angular/core";
```

Comment:
I prefer single quotes as well. It's just our reporter shows all the warnings in yellow for single quotes and white spaces while running test. So changed the code accordingly, thinking it might be a standard.